includes/Checker/Abstract_Check_Runner.php

```diff
@@ -346,4 +346,14 @@ public function get_plugin_basename() {
 
 		return $this->plugin_basename;
 	}
+
+	/** Gets the Check_Context for the plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return Check_Context The check context for the plugin file.
+	 */
+	protected function get_check_context() {
```

Comment:
Better to make this private since it's not needed in child classes (related to #162).